src/wp-includes/deprecated.php

```diff
@@ -6387,3 +6387,42 @@ function wp_enqueue_global_styles_custom_css() {
 		wp_add_inline_style( 'global-styles', $custom_css );
 	}
 }
+
+/**
+ * Loads classic theme styles on classic themes in the editor.
+ *
+ * This is used for backwards compatibility for Button and File blocks specifically.
+ *
+ * @since 6.1.0
+ * @since 6.2.0 Added File block styles.
```

Comment:
```suggestion
```

Not present in function docs `src/wp-includes/script-loader.php`.

Reply:
The second `since` tag is not present yet, but adding the File blocks styles probably should have been noted there already in 6.2.

I also edited the function description to mention File blocks and to replace "needed" with "used" (because the function would not be necessary if the stylesheet is enqueued in `enqueue_block_assets` instead).

Reply:
`wp_enqueue_classic_theme_styles()` docs needed an update more than this function (and I kept these changes too)